x/ibc/04-channel/exported/exported.go

```diff
@@ -1,13 +1,21 @@
 package exported
 
-// PacketI defines the standard interface for IBC packets
+// PacketDataI defines the standard interface for IBC packets
 type PacketI interface {
 	GetSequence() uint64
 	GetTimeoutHeight() uint64
 	GetSourcePort() string
 	GetSourceChannel() string
 	GetDestPort() string
 	GetDestChannel() string
-	GetData() []byte
+	GetData() PacketDataI
 	ValidateBasic() error
 }
+
+type PacketDataI interface {
```

Comment:
add godoc

Reply:
Yes please @mossid! All interfaces should have comments that describe their usage for Godoc.